### ISSUE TITLE
No Issues found with large Function Groups

### ISSUE THREAD

**Issue by OberstVonGatow** [abaplint/abaplint-sci-client#282]
Unfortunately we have a large Function Group (14 Function Modules) from a external Consulting Company (so refactoring everything would be no option right now). But we would like to check every new development with abaplint.
But we get zero issues but would expect over a hundred.

```
2023-04-21 12:51:14, listRules
2023-04-21 12:53:52, check_file, FUGR, ZINBOX_0, 0 issues, 2023834 bytes, 12035 ms
2023-04-21 12:53:52, ping
```

Log seems like everything is send correctly.
Version abaplint: 2.97.12

**Comment by larshp** [comment]
moved to abaplint-sci-client
